apps/financial-aid/web-osk/src/components/FileUploadResult/FileUploadResult.tsx

```diff
@@ -18,14 +19,22 @@ interface Props {
   children?: React.ReactNode
 }
 
-const FileUploadSuccess = ({
+const FileUploadResults = ({
   subtitle,
   subtitleColor,
   nextButtonText,
   nextButtonAction,
   children,
 }: Props) => {
-  const { form } = useContext(FormContext)
+  const router = useRouter()
+
+  const { form, emptyFormProvider } = useContext(FormContext)
+
+  useEffect(() => {
+    router.events.on('routeChangeComplete', () => {
+      emptyFormProvider()
```

Comment:
Good point, however im not sure that it matters, since we are not using the form provider for anything else at this moment in client. What do you think?

Reply:
Think it is more clear what is happening if we do: 
` updateForm({ ...form, otherFiles: [] })`
Because that is really the only thing we want to update.... right?